default.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":pinAllExceptPeerDependencies",
+    ":separateMajorReleases",
+    "config:base",
+    "group:monorepos",
+    "schedule:weekends",
+    "github>bitwarden/renovate-config:pin-actions"
+  ],
+  "commitMessagePrefix": "[deps]:",
+  "commitMessageTopic": "{{depName}}",
+  "prConcurrentLimit": 0,
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": [
+      "every 2nd week starting on the 2 week of the year before 4am on Monday"
+    ]
+  },
+  "schedule": [
+    "every 2nd week starting on the 2 week of the year before 4am on Monday"
```

Comment:
This will probably break in 2026 when we have 53 weeks and our release sprints are shifted a week and/or we get a 1 week / 3 week sprint to compensate.